docs/api/data.rst

```diff
@@ -48,6 +48,42 @@ It provides a set of 25,000 highly polar movie reviews for training, 25,000 for
 
     IMDB
 
+Sentiment Analysis: MR
+~~~~~~~~~~~~~~~~~~~~~~~~
+`MR <https://www.cs.cornell.edu/people/pabo/movie-review-data/>`_ is a data set of movie reviews for sentiment analysis. Available are collections of movie-review documents labeled with respect to their overall sentiment polarity (positive or negative).
```

Comment:
`MR <https://www.cs.cornell.edu/people/pabo/movie-review-data/>`_ is a movie-review data set of 10,662 sentences labeled with respect to their overall sentiment polarity (positive or negative).